main.c

```diff
@@ -69,6 +69,22 @@ int main(void)
     MSS_TIM64_enable_irq();
     NVIC_SetPriority(Timer1_IRQn, 1);
 
+    /* load citi conf on startup */
+    uint32_t cfg_id_32 = 0;
```

Comment:
Just wondering: Why "_32"? (@anwestm, I guess this question is more directed to you.)

Reply:
mem_read takes a uint32_t pointer

Reply:
I see. It should also work if you do `mem_read(NVM_CITIROC_CONF_NUM, (uint32_t *)&cfg_id);` on the next line. Might as well do that and test it -- if you intend to do some testing during the next couple of weeks.

Reply:
Fixed as part of commit 60f58db and tested to work.